libmscore/element.h

```diff
@@ -320,6 +327,8 @@ class Element : public QObject, public ScoreElement {
             ELEMENT_LIST,
             STAFF_LIST,
             MEASURE_LIST,
+
```

Comment:
there are lots of unnecessary extra blank lines added, maybe some of them might help with readability (I'm guessing the were more likely mistakes, though) - but two in a row should never happen